Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2006-01-02
Initial Package Version: 1.4.10
Origin: RockLinux and Jim Gifford
Upstream Status: Unknown
Description: Allows Silo to Build as 64-Bit
 
diff -Naur silo-1.4.9.orig/common/divdi3.S silo-1.4.9/common/divdi3.S
--- silo-1.4.9.orig/common/divdi3.S	2004-06-20 17:16:32.000000000 +0000
+++ silo-1.4.9/common/divdi3.S	2005-08-06 19:50:06.000000000 +0000
@@ -17,6 +17,9 @@
 the Free Software Foundation, 59 Temple Place - Suite 330,
 Boston, MA 02111-1307, USA.  */
 
+	.register %g3,#scratch
+	.register %g2,#scratch
+ 
 	.data
 	.align 8
 	.globl	__clz_tab
diff -Naur silo-1.4.9.orig/common/udivdi3.S silo-1.4.9/common/udivdi3.S
--- silo-1.4.9.orig/common/udivdi3.S	2004-06-20 17:16:32.000000000 +0000
+++ silo-1.4.9/common/udivdi3.S	2005-08-06 19:50:06.000000000 +0000
@@ -17,6 +17,8 @@
 the Free Software Foundation, 59 Temple Place - Suite 330,
 Boston, MA 02111-1307, USA.  */
 
+	.register %g2,#scratch
+
 	.text
 	.align 4
 	.globl __udivdi3
diff -Naur silo-1.4.9.orig/second/muldi3.S silo-1.4.9/second/muldi3.S
--- silo-1.4.9.orig/second/muldi3.S	2001-06-06 17:04:50.000000000 +0000
+++ silo-1.4.9/second/muldi3.S	2005-08-06 19:50:06.000000000 +0000
@@ -17,6 +17,8 @@
 the Free Software Foundation, 59 Temple Place - Suite 330,
 Boston, MA 02111-1307, USA.  */
 
+	.register %g2,#scratch
+
 	.text
 	.align 4
 	.globl __muldi3
